File: Pemrograman1/Modul4/C/PRAK405-2310817310009-MuhammadFauzanAhsani.c
#include <stdio.h>

int main(){
    int val1, val2;
    scanf("%d %d", &val1, &val2);

    for(int i = 1; i <= val1; i++){
        for(int j = 1; j <= i; j++){
            if (i == j){
                printf("(%d * %d) = ", i-j, val2);
            }
            else {
                printf("(%d * %d) + ", i-j, val2);
            }
        }
        printf("%d\n", (i * (i + 1) /2) * val2);
    }
    printf("%d", (val1 * (val1+1) * (val1+2)) * val2 / 6);
    return 0;    
}

File: Pemrograman1/Modul4/Python/PRAK405-2310817310009-MuhammadFauzanAhsani.py
var = input().split()
val1, val2 = map(int, var)

for i in range(1, val1 + 1):
    for j in range(i):
        if (i == j + 1):
            print(f"({i-j} * {val2}) = ", end = "")
        else:
            print(f"({i-j} * {val2}) + ", end ="")
    print(f"{(i * (i + 1) * val2 / 2):.0f}")
print(f"{(val1 * (val1 + 1) * (val1 + 2) * val2 / 6):.0f}")
